user/running-build-in-debug-mode.md

```diff
@@ -11,10 +11,19 @@ significant differences between your local development environment and
 the Travis CI build environment, you can restart builds in debug mode
 to get shell access to the virtual machine or container.
 
+## Enabling debug mode
+
+Private repositories have debug mode enabled by default, and no changes need to be made.
+To limit access to debug, grant users only *read access* to the repo, and use a fork + PR workflow.
+For public repositories, we have to enable it on a repository basis.  
+To enable debug for your public repositories, please email us at
+support@travis-ci.com and let us know which repositories you want activated.
+
 ## Restarting a job in debug mode
 
 The "Debug build" or "Debug job" button is available on the upper right corner of
-the build and job pages.
+the build and job pages for private repositories. For open source repositories, 
+this button is not available.
```

Comment:
Maybe mention that, in this case, the debug build needs to be started via an API call?